### PR TITLE
スマホ対応

### PR DESCRIPTION
**Pull request by suzan2go**
* スマホ対応です。
* bootstrap-railsのbootstrapのversionが低いままなので、捨ててbootstrap-sassにします

### CODE REVIEW

File: app/helpers/bootstrap_flash_helper.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+module BootstrapFlashHelper
```
Comment (nasum) [inline app/helpers/bootstrap_flash_helper.rb]: __[rubocop]__
* Missing top-level module documentation comment.

File: app/helpers/bootstrap_flash_helper.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  ALERT_TYPES = [:success, :info, :warning, :danger] unless const_defined?(:ALERT_TYPES)
```
Comment (nasum) [inline app/helpers/bootstrap_flash_helper.rb]: __[rubocop]__
* Line is too long. [88/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: app/helpers/bootstrap_flash_helper.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  def bootstrap_flash(options = {})
```
Comment (nasum) [inline app/helpers/bootstrap_flash_helper.rb]: __[rubocop]__
* Assignment Branch Condition size for bootstrap_flash is too high. [22.74/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Cyclomatic complexity for bootstrap_flash is too high. [7/6]
* Method has too many lines. [19/10] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)

File: app/views/home/index.html.erb [hunk 4]
```diff
@@ -18,22 +18,26 @@
...
-      </div>
-      <div>
-        <%= poem.user.name %>
-      </div>
-      <div class="text-right">
-        <%= poem.created_at %>
+      <li class="list-group-item col-md-4">
+        <div class="medila">
+          <div class="media-left">
+            <%= image_tag(poem.user.image_url, class: "img-circle", width: 60) %>
```
Comment (nasum) [inline app/views/home/index.html.erb]: __[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)

File: app/helpers/bootstrap_flash_helper.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    flash_messages = []
+    flash.each do |type, message|
+      # Skip empty messages, e.g. for devise messages set to nothing in a locale file.
```
Comment (nasum) [inline app/helpers/bootstrap_flash_helper.rb]: __[rubocop]__
* Line is too long. [86/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: app/views/home/index.html.erb [hunk 6]
```diff
@@ -18,22 +18,26 @@
...
+          </div>
+          <div class="media-body">
+            <div class="media-heading">
+              <%= link_to poem.title.truncate(15), poem %>
+            </div>
+            <div class="media-content">
+              <%= poem.user.name %>
```
Comment (nasum) [inline app/views/home/index.html.erb]: __[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)

File: app/helpers/bootstrap_flash_helper.rb [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+      type = :danger  if type == :alert
+      type = :danger  if type == :error
+      next unless ALERT_TYPES.include?(type)
+
+      tag_class = options.extract!(:class)[:class]
+      tag_options = {
+        class: "alert fade in alert-#{type} #{tag_class}"
+      }.merge(options)
+
+      close_button = content_tag(:button, raw("&times;"), type: "button", class: "close", "data-dismiss" => "alert")
```
Comment (nasum) [inline app/helpers/bootstrap_flash_helper.rb]: __[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Line is too long. [116/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

**Comment by suzan2go** [comment]
bootstrap_flashはbootstrap-railsのhelperで使っていたもののをそのまま移植してます

**Comment by nasum** [comment]
[LGTM](http://lgtm.in/i/IeILuSo6q)

**Pull request merged**